mapbook-iOS.xcodeproj/project.pbxproj

```diff
@@ -464,24 +501,18 @@
 				CODE_SIGN_STYLE = Automatic;
 				DEVELOPMENT_TEAM = "";
 				FRAMEWORK_SEARCH_PATHS = (
-					"$(inherited)",
-					"$(USER_LIBRARY_DIR)/SDKs/ArcGIS/iOS/Frameworks/Static",
+					$HOME/Library/SDKs/ArcGIS/iOS/Frameworks/Dynamic/,
```

Comment:
You need either this or the following line, but not both. They both point to the same place. Same with lines `527` and `528`.

Reply:
Xcode added the next line on its own.